answer.txt

```diff
@@ -0,0 +1,11 @@
+• What is a gitignore file and what should it include for a Java project?
+	So the gitignore file contains the ignore rules. Place the gitignore file in the main directory of the Eclipse Project.
```

Comment:
The question asks for what should be included in the `.gitignore` for a java project, your answer doesn't mention that. Take a look at the sample `.gitingore` added to your repo from selecting java project (as per the assignment)

Reply:
> So

Answer should be formal